components/Compare/Compare.js

```diff
@@ -25,6 +25,15 @@ const StyledNameCompareH4 = styled.h4`
   font-size: 1.2em;
 `;
 
+const StyledYourCarDiv = styled.div`
+  /* background-color: transparent;
+  border: none;
+  display: flex;
+  position: relative; */
+  font-size: 1.5em;
+  margin-bottom: 13px;
+`;
+
```

Comment:
try not to merge commented out code:

```suggestion
const StyledYourCarDiv = styled.div`
  font-size: 1.5em;
  margin-bottom: 13px;
`;

```

Reply:
Thank you very much!!!